### DISCUSSION TITLE
RustDesk Server Pro --> Relay to OSS Server

### DISCUSSION THREAD

**Discussion by ekunzag** [rustdesk/rustdesk#8156] [Q&A]
Hello

We bought RustDesk Server Pro licenses and we wonder if there is a way to manage following:

If we used all agents from our pro license, could we setup a relay server to a RustDesk Server OSS version for the agents, which we don't want to use over our pro server? Or like if there is no agent license left, could a relay to the RustDesk Server OSS work?

We don't want to use every agent over the pro server, so is there a way to get this to work?

Thanks for the answers
Andrin Gerber

**Comment by rustdesk** [comment]
https://x.com/rustdesk/status/1721799106698396029?t=y5zXt0Oqt4rHJyn7YkTiwQ&s=19

**Reply by ekunzag** [reply]
Thank you for the answer. I now struggle to get the URL Action for Tactical RMM working.
The URL Action: rustdesk://connection/new/{{agent.rustdeskid}}{{global.rustdesk.test}}?password={{agent.rustdeskpwd}}
The "global.rustdesk.test" is a Key Store value: "@<my_server>?key=<my_key>"
The Key name is "rustdesk.test", so it would be "global.rustdesk.test" as I understand.

RustDesk Client says that the ID doesn't exist. With the 1.2.4 version when I put in the command manually, it works fine.

Can you help me with that?

**Reply by ekunzag** [reply]
It worked now to get the output I want. But it always says the key isn't the same. I assume that RustDesk doesn't use the "<id>@<my_server>?key=<my_key>" for the URL Action the same way as in the program itself. Because in the program it works perfectly but over the browser with rustdesk:// or Action URL it doesn't work with the exact same values :(
